test/common/__snapshots__/graph.js.snap

```diff
@@ -184,11 +177,10 @@ exports[`graph #visitor should visit a model manager and create a dependency gra
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`concerto@1.0.0.Transaction\`
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`org.acme.hr@1.0.0.Person\`
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`org.acme.hr.base@1.0.0.Address\`
+   \`\`
+   \`\` --> \`org.acme.hr.base@1.0.0.GeneralCategory\`
    \`org.acme.hr@1.0.0.KinName\`
-   \`org.acme.hr@1.0.0.KinName\` --> \`org.acme.hr.base@1.0.0.GeneralCategory\`
    \`org.acme.hr@1.0.0.KinTelephone\`
-   \`org.acme.hr@1.0.0\`
-   \`org.acme.hr@1.0.0\` --> \`org.acme.hr.base@1.0.0.GeneralCategory\`
```

Comment:
Is it expected that this edge is removed?

Reply:
No. Adding this back.